blocks/bricks/__init__.py

```diff
@@ -444,6 +444,32 @@ class Softmax(Activation):
     def apply(self, input_):
         return tensor.nnet.softmax(input_)
 
+    @application
+    def categorical_cross_entropy(self, x, y):
```

Comment:
I was just using this method, and realised that the arguments are in the reverse order from `CategoricalCrossEntropy`. It would be nicer if they are the same i.e. `self, y, x`. Could you make that change?